chat-plugins/github.js

```diff
@@ -0,0 +1,94 @@
+/*
+	Github plugin by Spandamn
+	This is based on https://github.com/Ecuacion/Pokemon-Showdown-Node-Bot/blob/master/features/github/index.js
+	Yes, some of the code is copied so credits to xfix, Ecuacion, jd, and anyone who's name I forgot to put here
```

Comment:
Blah, I think "some" is kind of an understatement here...

jd was the one who originally converted this to be server-side, and I only see 3 subtle changes from that version than the original proposed one: https://github.com/jd4564/Pokemon-Showdown/blob/master/github.js